Documentation/network/kubernetes/compatibility-table.rst

```diff
@@ -110,7 +114,9 @@
 +--------------------+----------------+
 | v1.14.4            | 1.26.11        |
 +--------------------+----------------+
-| v1.14              | 1.26.11        |
+| v1.14.5            | 1.26.11        |
++--------------------+----------------+
+| v1.14              | 1.27.0         |
 +--------------------+----------------+
-| latest / main      | 1.26.12        |
+| latest / main      | 1.29.0         |
```

Comment:
I'm surprised to see `1.29.0` here, given that it's `1.26.12` in `main`. Also I'd expect the same value to be used for `latest / main` in the three release PRs? Or am I mistaken?

Reply:
OK, `1.29.0` was recently added in 9b016904f8aa ("Prepare for v1.16 development cycle"), so I think it's the right value.

The sync script probably needs an update to use `main` on the oldest branches, but we can address that as a follow-up I think. Thanks for checking!

Reply:
ohh now I got what the problem is. Yes, it should be 1.29.0 everywhere but it's not a reason to block this PR.